src/lib/utils_server.ts

```diff
@@ -21,6 +21,7 @@ export type AddResult = {
 	added: boolean;
 }
 
+export type processResult = { success: boolean; message: string; domain: string; selector: string }[] | { processResultError: string; } | undefined;
```

Comment:
All type names in this project are capitalized, i.e. this should be ProcessResult.
I was actually a bit confused for a while when reviewing the code and thought I was looking at a variable and not a type, so stuff like does matter :)